docs/development/release.md

```diff
@@ -80,7 +79,7 @@ versions to that.
 Manually create a release on github & upload, but soon we'll publish shipbot which automates this...
```

Comment:
Just published it.  Will update the docs in a separate PR so I don't have to rebuild alpha.1 :-)